src/Microsoft.AspNet.Mvc.DataAnnotations/DataTypeClientModelValidatorProvider.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.AspNet.Mvc.ModelBinding.Validation
+{
+    /// <summary>
+    /// An implementation of <see cref="IClientModelValidatorProvider"/> which provides client validators
+    /// based on the model type.
```

Comment:
Is this comment intended to suggest you think we should follow up with a change to get the value `Type` -> validator `Type` mapping from `MVCConfig`? Either way, it's odd to have the documentation imply broader aspirations rather than what the provider does today. Today this provider provides `NumberClientModelValidator`s for a few `Type`s not covered already.